Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -196,8 +196,68 @@ import BraintreeDataCollector
         _ request: BTPayPalVaultEditRequest,
         completion: @escaping (BTPayPalVaultEditResult?, Error?) -> Void
     ) {
-        // TODO: call API to get FI URL and return a BTPayPalVaultEditResult or Error
-        completion(nil, nil)
+        // TODO: Analytics event
+
+        apiClient.fetchOrReturnRemoteConfiguration { configuration, error in
+            if let error {
+                self.notifyEditFIFailure(with: error, completion: completion)
+                return
+            }
+
+            guard let configuration, let json = configuration.json else {
+                self.notifyEditFIFailure(with: BTPayPalError.fetchConfigurationFailed, completion: completion)
+                return
+            }
+
+            self.isConfigFromCache = configuration.isFromCache
+
+            guard json["paypalEnabled"].isTrue else {
+                self.notifyEditFIFailure(with: BTPayPalError.disabled, completion: completion)
+                return
+            }
```

Comment:
I ran into issues earlier extracting this into common function with subclassing or with protocols because of the difference in riskCorrelationID property. I got errors regarding ambiguous context.